src/shared/types/Styles.ts

```diff
@@ -0,0 +1,10 @@
+export type DirectionUnionType = "back" | "forward";
+
+export interface StylePropType {
+  // Left stype parametr
```

Comment:
Please use this extention to fix english words:
streetsidesoftware.code-spell-checker